common/app/experiments/Experiments.scala

```diff
@@ -30,7 +30,7 @@ object DCRFronts
     extends Experiment(
       name = "dcr-fronts",
       description = "Use DCR for fronts",
-      owners = Seq(Owner.withGithub("dotcom")),
+      owners = Seq(Owner.withGithub("@guardian/dotcom-platform")),
```

Comment:
We use a variety of:
- `SwitchGroup` names
- team emails in `*@guardian.co.uk` and `*@theguardian.com`
- Github team handles

I’m not entirely sure of the benefits of having so many options.